tests/props.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { mount } from '../src'
+import WithProps from './components/WithProps.vue'
+
+describe('props', () => {
+  it('returns a single prop applied to a component', () => {
+    const wrapper = mount(WithProps, { props: { msg: 'ABC' } })
+    expect(wrapper.props('msg')).toEqual('ABC')
+  })
+
+  it('returns all props applied to a component', () => {
+    const wrapper = mount(WithProps, { props: { msg: 'ABC' } })
+    expect(wrapper.props()).toEqual({ msg: 'ABC' })
+  })
+})
```

Comment:
Should we add a test to document what happens if the requested prop is not there, or if the target component has no props?

Reply:
Done.